Makefile

```diff
@@ -294,6 +294,31 @@ build-install:
 	cp ./target/release/wasmer ./install/bin/
 	tar -C ./install -zcvf wasmer.tar.gz bin/wapm bin/wasmer
 
+UNAME_S := $(shell uname -s)
+
+build-capi:
```

Comment:
It seems that `build-capi` doesn't actually build the capi, it assumes it's there and copies it.  I think we should rename it!